maven-wrapper-distribution/src/resources/mvnw.cmd

```diff
@@ -160,6 +160,7 @@ FOR /F "usebackq tokens=1,2 delims==" %%A IN ("%MAVEN_PROJECTBASEDIR%\.mvn\wrapp
 )
 IF NOT %WRAPPER_SHA_256_SUM%=="" (
     powershell -Command "&{"^
+       "Import-Module $PSHOME\Modules\Microsoft.PowerShell.Utility -Function Get-FileHash"^
```

Comment:
I assume that a `;` is needed at the end of Get-FileHash.

Reply:
Hm, no idea, but Google found me this: https://poshcode.gitbook.io/powershell-practice-and-style/style-guide/code-layout-and-formatting#avoid-using-semicolons-as-line-terminators

Reply:
even if it is to avoid, will add to be consistent with existing PS code....